charts/prometheus/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
-appVersion: v2.55.0
-version: 25.29.0
+appVersion: v2.55.1
+version: 25.29.1
```

I reckon we should bump the minor number
```suggestion
version: 25.30.0
```